02_tidy-evaluation-metaprogramming.Rmd

```diff
@@ -317,6 +316,63 @@ my_pivot_longer <- function(data, ...) {
 mtcars %>% my_pivot_longer(cyl, am = am * 100)
 ```
 
+### Double Evaluation {-}
```

Comment:
I forget exactly how bookdown works, is it ## for a new page altogether?

Reply:
Yup!

Reply:
```suggestion
## Guide: Double Evaluation {-}
```